enhancements/README.md

```diff
@@ -0,0 +1,11 @@
+# Harvester Enhancement Proposal (HEP)
+
+This directory serves as a collection of Harvester Enhancement Proposals (HEPs). HEPs are used to describe highlighted features, enhancements, or bug fixes that require substantial changes to the code or architecture. To create a HEP, you can use the provided [template](./YYYYMMDD-template.md) and submit a pull request.
+
+The following steps describe how HEPs are involved in the development process:
+
+1. An engineer or contributor is assigned a GitHub issue. The GitHub issue should have the `require/HEP` label.
```

Comment:
Can add 
`If the issue doesn't have require/HEP initially but after the investigation, the assigned engineer or contributor thinks it's preferred to have a HEP, they can add the label or contact the project's maintainer to add the label.`

Reply:
Added, thanks!